pkg/models/system.go

```diff
@@ -0,0 +1,37 @@
+package models
+
+import (
+	"time"
+
+	"github.com/google/uuid"
+)
+
+// System represents something that may be tested for 508 compliance
+// TODO: if we are using the `SystemFromIntake` strategy, the "db" field tags would be unecessary;
+// but if we are trying to read the System type straight from sqlx we WOULD need the "db" field tags
+type System struct {
+	IntakeID    uuid.UUID  `json:"intakeId" db:"id"` // TODO: is this actually necessary, if LCID is really the identifier?
+	LCID        string     `json:"lcid" db:"lcid"`
+	CreatedAt   *time.Time `json:"createdAt" db:"created_at"`
+	UpdatedAt   *time.Time `json:"updatedAt" db:"updated_at"`
+	IssuedAt    *time.Time `json:"issuedAt" db:"decided_at"`
+	ExpiresAt   *time.Time `json:"expiresAt" db:"lcid_expires_at"`
+	ProjectName string     `json:"projectName" db:"project_name"`
+	OwnerID     string     `json:"ownerId" db:"eua_user_id"`
+	OwnerName   string     `json:"ownerName" db:"requester"` // TODO: wouldn't really be necessary at DB layer if we had performant access to LDAP
+}
+
+// SystemFromIntake builds a System type from its constituent SystemIntake
+func SystemFromIntake(intake *SystemIntake) *System {
+	return &System{
+		IntakeID:    intake.ID,
+		LCID:        intake.LifecycleID.ValueOrZero(),
+		CreatedAt:   intake.CreatedAt,
+		UpdatedAt:   intake.UpdatedAt,
+		IssuedAt:    intake.DecidedAt,
+		ExpiresAt:   intake.LifecycleExpiresAt,
+		ProjectName: intake.ProjectName.ValueOrZero(),
+		OwnerID:     intake.EUAUserID.ValueOrZero(),
+		OwnerName:   intake.Requester,
+	}
+}
```

Comment:
This func is speculative... Maybe or may not be needed, based on whether or not we can get all the required filtering done within the SQL query. (See the `storage` changes.)